src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task {
+
+    protected String dueDate;
+    Deadline(String taskName, String dueDate) {
+        super(taskName);
+        this.dueDate = dueDate;
+    }
+    @Override
+    public void printTaskType() {
+        System.out.print("D");
+    }
+    @Override
+    public void printTask() {
+        System.out.print("[");
+        printTaskType();
+        System.out.print("][");
+        if (super.isDone) {
+            System.out.print("X");
+        } else {
+            System.out.print(" ");
+        }
+        System.out.println("] " + this.getTaskName() + "(by:" + dueDate + ")");
```

Comment:
Might there be a better way to print these information? Constants and concatenation